src/validate.py

```diff
@@ -144,3 +125,44 @@ def parse_accept_language(header, supported_langs):
 
     # If all else fails, default the language.
     return DEFAULT_LANGUAGE.lang_code
+
+def get_json_files(path):
+
+    files_found = []
+    for root, directories, files in os.walk(path):
+        for file in files:
+            if '.json' in file:
+                files_found.append(os.path.join(root, file))
+
+    return files_found
+
+def get_entries_from_json(path, p_key, s_key):
+
+    with open(path) as json_file:
+        data = json.load(json_file)
+
+    entries = []
+
+    if p_key in data:
+        for values in data.get(p_key):
+             entries.append(values.get(s_key))
+
+    return entries
+
+def get_chapters(file):
+
+    chapters = []
+
+    data = get_entries_from_json(file,'outline','chapters')
+    for list in data:
+        for entry in list:
+            chapters.append(entry.get('title').lower().replace(" ", "-").replace("/",""))
+
+    return chapters
+
+json_files = get_json_files(CONFIG_DIR)
```

Comment:
Similarly, could this be capitalised as it is module scoped or perhaps named a little more meaningfully, such as `YEAR_CONFIG_FILES` or something.

Reply:
sure